clickhouse-http-client/src/main/java/com/clickhouse/client/http/ApacheHttpConnectionImpl.java

```diff
@@ -115,7 +115,8 @@ private ClickHouseHttpResponse buildResponse(ClickHouseConfig config, CloseableH
         // {"read_rows":"0","read_bytes":"0","written_rows":"0","written_bytes":"0","total_rows_to_read":"0"}
         String displayName = getResponseHeader(response, "X-ClickHouse-Server-Display-Name", server.getHost());
         String queryId = getResponseHeader(response, "X-ClickHouse-Query-Id", "");
-        String summary = getResponseHeader(response, "X-ClickHouse-Summary", "{}");
+        Header hSum = response.getLastHeader("X-ClickHouse-Summary");
```

Comment:
I will revert this.

Reply:
- I've extracted to ClickHouseHttpProto class the constants 
- I've created method reading headers in the new client helper 
- I've reverted this change because it doesn't make sense.